tripy/tests/integration/test_layernorm.py

```diff
@@ -67,6 +67,7 @@ def test_layernorm_improper_dimensions(self):
         )
         x = tp.ones((5,5,5))
         with pytest.raises(
```

Comment:
Changed to `helper.raises`, but the error message could not trace back all the way to input, because layernorm is a composite module:
```
--> /tripy/tripy/frontend/module/layernorm.py:115 in __call__()
          |
      115 |         return self.weight * x + self.bias
          |                ^^^^^^^^^^^^^^^

    Required from:
    --> /tripy/tests/integration/test_layernorm.py:74 in test_layernorm_improper_dimensions()
          |
       74 |             tp_layernorm(x).eval()
          |

    Input 0:

    --> /tripy/tripy/frontend/trace/ops/reshape.py:146 in reshape()
          |
      146 |     return reshape_impl(input, shape, len(shape), output_len)
          |

    --> /tripy/tripy/frontend/module/parameter.py:78 in __init__()
          |
       78 |         super().__init__(reshape(arange(utils.volume(shape), dtype), shape))
          |                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ --- required from here

    Required from:
    --> /tripy/tripy/frontend/module/layernorm.py:92 in __init__()
          |
       92 |         self.weight = DefaultParameter(normalized_shape, dtype=dtype)
          |

    Required from:
    --> /tripy/tripy/utils/utils.py:244 in custom_init()
          |
      244 |             return default_init(self, *args, **kwargs)
          |

    Required from:
    --> /tripy/tests/integration/test_layernorm.py:65 in test_layernorm_improper_dimensions()
          |
       65 |         tp_layernorm = tp.LayerNorm(
          |

    Input 1:

    --> /tripy/tripy/frontend/module/layernorm.py:114 in __call__()
          |
      114 |         x = (x - mean_val) * rsqrt(var_val)
          |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

    Required from:
    --> /tripy/tests/integration/test_layernorm.py:74 in test_layernorm_improper_dimensions()
          |
       74 |             tp_layernorm(x).eval()
          |
```

Reply:
Ah I see, that's fine then.